src/locales/cn/translation.json

```diff
@@ -16,9 +16,9 @@
       "merchants": "WAYS TO DONATE",
       "waystodonate": {
         "donate": "DONATE",
+        "light-up-chinatown": "LIGHT UP CHINATOWN",
```

Comment:
oops but it looks like it's supposed to be a main navigation: https://www.sendchinatownlove.com/